lib/dateDisplay.ml

```diff
@@ -491,26 +491,51 @@ let string_slash_of_date conf date =
       " (" ^ transl_nth conf "gregorian/julian/french/hebrew" 3 ^ ")"
   | Dtext t -> t
 
+let negative_age a =
+  let _ = Printf.eprintf "Year: %d, month: %d, day: %d\n" a.year a.month a.day in
+  let ny = a.year + 1 in
+  let nm = if a.day = 0 then a.month - 12 else a.month - 11 in
+  let nd = a.day - 30 in
+  let _ = Printf.eprintf "Nyear: %d, nmonth: %d, nday: %d\n" ny nm nd in
+  {day = nd; month = nm; year = ny; prec = Sure; delta = 0;}
+
+(* year may be <0, but not month or days *)
+(* -3 days is year=-1;month=11;days=28   *)
```

Comment:
`-3 days is year=-1;month=11;days=28` correspond a l'état actuel des choses, si j'ai bien suivi ?

Reply:
Oui, c'est le format actuel fournit à `string_of_age` dans les cas négatifs.